Mortgage-loan-calculator/src/app/calculator-form/calculator-form.component.html

```diff
@@ -1,329 +0,0 @@
-<head>
-  <script src="./show.js"></script>
-</head>
-<div class="formOuter">
-  <div class="formBlock">
-    <h2>Mortgage loan calculator</h2>
-
-    <div class="orange">
-      <div class="box">
-        <mat-tab-group>
-          <mat-tab routerLink="/borrow" label="How much can you borrow">
-            <ng-template matTabContent>
-              <div class="columns">
-                <form
-                  class="column"
-                  [formGroup]="calculateForm"
-                  (ngSubmit)="onSubmit()"
-                  (change)="onSubmit()"
```

Comment:
Form value changes should be observed using `calculateForm.valueChanges`in component.